src/main/java/nl/tudelft/dnainator/Main.java

```diff
@@ -1,7 +1,8 @@
 package nl.tudelft.dnainator;
 
-import javafx.application.Application;
+import com.sun.javafx.application.LauncherImpl;
```

Comment:
If you can find another way to attach the preloader, please tell me because I couldn't. :(